template.yaml

```diff
@@ -324,6 +328,26 @@ Resources:
             Type: SQS
             Destination: !GetAtt DataLoadingDLQ.Arn
 
+
+  BulkDataLoader:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: bulk-load
+      Policies:
+        - VPCAccessPolicy: { }
+      Handler: no.sikt.nva.data.report.api.etl.loader.BulkLoadHandler::handleRequest
+      Timeout: 600
+      MemorySize: 1536
+      AutoPublishAlias: live
+      Environment:
+        Variables:
+          LOADER_BUCKET: !Ref LoaderBucketName
+          NEPTUNE_ENDPOINT: !GetAtt NeptuneCluster.Endpoint
+          NEPTUNE_PORT: 8182
+          LOADER_IAM_ROLE: !Get ReadAndListBucketPolicy.Arn
```

Comment:
Looks rights, however chatgpt suggests to change 
`LOADER_IAM_ROLE: !Get ReadAndListBucketPolicy.Arn` 
to 
`LOADER_IAM_ROLE: !GetAtt ReadAndListBucketPolicy.Arn`